### ISSUE TITLE
Open Edirom with specific work context?

### ISSUE THREAD

**Issue by nikobeer** [Edirom/Edirom-Online#91]
How to?

**Comment by bwbohl** [comment]
you mean with a specific "activeWork" or did I get you wrong?
In case I didn't see: 
https://github.com/Edirom/Edirom-Online/wiki/Getting-Edirom-Online#configure

**Comment by nikobeer** [comment]
@bwbohl yes, back than I wanted to directly link to a specific work context via parameter. As this is possible now (the parameter is `workID`) this ticket is obsolete. Will close…

**Comment by bwbohl** [comment]
cf. https://github.com/Edirom/Edirom-Online/wiki/Links-to-Edirom-Online-and-its-content#link-to-edirom-online-setting-a-active-work
